torchci/lib/bot/cliParser.ts

```diff
@@ -99,6 +99,10 @@ branch_selection.add_argument("-s", "--stable", {
   action: "store_true",
   help: "Rebase to viable/strict",
 });
+branch_selection.add_argument("-m", "--master", {
```

Comment:
Just a thought, I wonder if we will eventually have a main branch instead of master

Reply:
Yes, I propose to change this option to `--onto` where one can specify a branch name (and default would redirect to master)

Reply:
--onto sounds good. We already have a `--branch` option that we can repurpose for that

Reply:
> Looks like it confuses the current semantic even further. If you want `--stable` to become default behavior, you can do so, but also, please make sure that we never rebase backwards (i.e. if current commit base is more resent that `viable/strict` we will refuse to rebase)

I don't believe rebasing backwards is possible, but I'll reconfirm

Reply:
Basically, the orig version would have been a shortcut for someone who really wants the master branch.

But that should be a rare enough occurrence that there's not real need to offer it.  `--onto` functionality already exists in the form of the `--branch` parameter and let's just stick with that one